uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -143,7 +143,10 @@ public class MqttControlSteps {
 
 
     /** Actual list of user properties to transmit. */
-    private static final List<Mqtt5Properties> txUserProperties = null;
+    private static List<Mqtt5Properties> txUserProperties = null;
```

Comment:
1. Where txUserProperties is used ?
2. Create DEFAULT_TX_USER_PROPERTIES equals to null

Reply:
txUserProperties is still not used to connect, subscribe, publish, unsubscribe, disconnect requests.